CONTRIBUTING.md

```diff
@@ -20,15 +20,6 @@ source ~/.bash_profile
 
 You may also have to install "Android SDK Command Line Tools" from the SDK tools screen in Android Studio.
 
-Install system images
```

Comment:
I moved this down to where the emulator is created as I always missed this and the other command failed because the emulator was missing